packages/stargate/src/queries/utils.ts

```diff
@@ -9,11 +12,31 @@ export function toAccAddress(address: string): Uint8Array {
   return Bech32.decode(address).data;
 }
 
-/**
- * Use this to convert a protobuf.js class to the interface (e.g. Coin to ICoin)
- * in a ways that makes Jasmine's toEqual happy.
- */
-// eslint-disable-next-line @typescript-eslint/ban-types
-export function toObject<I extends object>(thing: I): Omit<I, never> {
-  return { ...thing };
+export function createPagination(
+  paginationKey?: Uint8Array,
+): {
+  readonly key: Uint8Array;
+  readonly offset: Long;
+  readonly limit: Long;
+  readonly countTotal: boolean;
+} {
```

Comment:
As mentioned above, this can probably return `undefined` of `paginationKey` is unset